### PR TITLE
Feature version mobile

### PR DESCRIPTION
**Pull request by LuisFerRG**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,24 +1,40 @@
...
-            <form> <input type="email">
-            </form> <br> <br>
-            <form> <input type="submit" value="Sign me up">
-            </form>
-            
-        </p>
+  <div class="container">
+    <header class="fisherman">
+      
+        <h1><br>got marketing?<br>
```
Comment (Edjhernandez) [inline index.html]: Good job @LuisFerRG!!! 
In line 16, I suggest,  you can use <br> tag just at the end of the line where you want to force line breaks.

File: index.html [hunk 2]
```diff
@@ -1,24 +1,40 @@
...
+      </header>
+      <section class="content">
+
+      <div class="fish-hook1">
+        <h2>Name</h2>
+        <input type="name">
+      </div>
+
+      <div class="fish-hook2">
+        <h3>Email</h3>
```
Comment (Edjhernandez) [inline index.html]: In this case, lines 24 and 29, as part of a form these tags have the same importance level, I suggest that using the same tag.

Comment (Edjhernandez) [reply index.html]: or label tag, you could investigate that.

File: styles.css [hunk 3]
```diff
@@ -1 +1,247 @@
...
+  font-family: Poppins, Open Sans, sans-serif;
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 27px;
+  letter-spacing: 0;
+  text-align: left;
+  margin: 14.94px 0;
+}
+
+.fish-hook1 > input[type="name"] {
```
Comment (Edjhernandez) [inline styles.css]: Remember DRY, you can use some selectors that have the same rules.
lines 73-79 and 93-99 have the same properties.

.fish-hook1 > input[type="name"],
.fish-hook2 > input[type="email"] {
   properties in common
}

File: styles.css [hunk 4]
```diff
@@ -1 +1,247 @@
...
+
+.container {
+  display: flex;
+  justify-content: center;
+  flex-flow: row wrap;
+  align-items: center;
+}
+
+.fisherman {
+  max-width: 100%;
```
Comment (Edjhernandez) [inline styles.css]: I suggest using 'px 'to fixed max-min

File: styles.css [hunk 5]
```diff
@@ -1 +1,247 @@
...
+  height: 100vh;
+  max-height: 576px;
```
Comment (Edjhernandez) [inline styles.css:15]: like this, great job...

File: styles.css [hunk 6]
```diff
@@ -1 +1,247 @@
...
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  justify-content: center;
+  align-items: center;
+  padding: 0 2em;
+}
+
+h1 {
+  padding: 0 3%;
```
Comment (Edjhernandez) [inline styles.css]: i suggest to investigate how units of % works with padding and marging

File: styles.css [hunk 7]
```diff
@@ -1 +1,247 @@
...
+  margin-top: 8%;
+  max-height: 563px;
+  justify-content: center;
+  flex-direction: column;
+  display: flex;
+  align-items: center;
+  padding: 0 2em;
+  margin-bottom: 2em;
+}
+
```
Comment (Edjhernandez) [inline styles.css]: From here, designs for other screens are included, but this PR says it is mobile only...

I suggest that you keep each screen design in different branches...

**Review by Edjhernandez — CHANGES_REQUESTED** [review]
Congrats @LuisFerRG !!!
you did a very good job, please check my comments...
Keep coding both!

**Comment by LuisFerRG** [comment]
Mobile screen 

image

**Pull request merged**